CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ if(POLICY CMP0064)
 endif()
 
 set(PROJECT_NAME "Cockatrice")
+set(PROJECT_VERSION_MAJOR "2")
+set(PROJECT_VERSION_MINOR "3")
+set(PROJECT_VERSION_PATCH "15")
+# Set to true to append git commit hash to version number
+set(PROJECT_IS_PRERELEASE True)
```

Comment:
How should we configure travis to do full releases? Right now we rely on the builds that CI produces to populate the release page, but with this the compiled ouput will say it's prerelease